File: module_login/src/main/java/debug/LoginApplication.java
package debug;

import com.ebook.common.BaseApplication;
import com.ebook.common.interceptor.LoginInterceptor;
import com.therouter.router.NavigatorKt;

public class LoginApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        NavigatorKt.addRouterReplaceInterceptor(new LoginInterceptor());
    }
}

File: module_login/src/main/java/debug/LoginApplication.kt
package debug

import com.ebook.common.BaseApplication
import com.ebook.common.interceptor.LoginInterceptor
import com.therouter.router.addRouterReplaceInterceptor

class LoginApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        addRouterReplaceInterceptor(LoginInterceptor())
    }
}
